estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,56 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+from datetime import timedelta
+
+
+class EstatePropertyOffer(models.Model):
+    _name = 'estate.property.offer'
+    _description = "Property Offer"
+    _order = 'price desc'
+    _price_check = models.Constraint('CHECK(price > 0)', "The offer price must be strictly positive.")
+
+    price = fields.Float()
+    partner_id = fields.Many2one('res.partner', string="Partner", required=True)
+    property_id = fields.Many2one('estate.property', string="Property", required=True)
+    property_type_id = fields.Many2one(related='property_id.property_type_id', string="Property Type", store=True)
+    validity = fields.Integer(string="Validity (days)", default=7)
+    date_deadline = fields.Date(string="Deadline", compute='_compute_date_deadline', inverse='_inverse_date_deadline')
+    status = fields.Selection(
+        selection=[('accepted', "Accepted"), ('refused', "Refused")],
+        copy=False
+    )
+
+    @api.depends('create_date', 'validity')
+    def _compute_date_deadline(self):
+        for record in self:
+            start_date = record.create_date.date() if record.create_date else fields.Date.today()
```

Comment:
Isn't `record.create_date` better here ? instead of `record.create_date.date()`.

```suggestion
            start_date = record.create_date or fields.Date.today()
```